src/Platform/Microsoft.Testing.Platform/DotnetTest/DotnetTestHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Testing.Platform.CommandLine;
+
+namespace Microsoft.Testing.Platform.DotnetTest;
+
+internal static class DotnetTestHelper
+{
+    private const string DotnetTestCliProtocol = "dotnettestcli";
```

Comment:
I would keep this one in the Platform defaults constants. So we have everything in one place when we check what we expose.